server/src/main/java/com/epam/aidial/core/server/security/AccessService.java

```diff
@@ -61,6 +61,15 @@ public boolean hasReadAccess(ResourceDescriptor resource, ProxyContext context)
         return permissions.get(resource).contains(ResourceAccessType.READ);
     }
 
+    public boolean hasWriteAccess(ResourceDescriptor resource, ProxyContext context) {
+        if (hasAdminAccess(context)) {
```

Comment:
Why do you need it? The code below will check it.

Reply:
Removed.